client/src/components/Summary.jsx

```diff
@@ -4,89 +4,104 @@
-          <span>Abode</span>
-          <IconButton icon={heartIcon} text="Save" />
-          <IconButton icon={shareIcon} text="Share" />
-          <IconButton icon={moreIcon} text="More" />
-        </div>
-        <div className="body">
-          <div className="specs">
-            <span>${summary.price}</span>
-            <span>{`${summary.numBD}bd`}</span>
-            <span> | </span>
-            <span>{`${summary.numBA}ba`}</span>
-            <span> | </span>
-            <span>{`${summary.sqft} sqft.`}</span>
+      <div className="summary">
+        <div className="main">
+          <div id="logo">Abode</div>
+          <div className="header">
+            <IconButton icon={heartIcon} text="Save" handleClick={() => this.showModal('signin')} />
+            <IconButton icon={shareIcon} text="Share" handleClick={() => this.showModal('share')} />
           </div>
-          <div className="address">
-            {summary.address}
-          </div>
-          <div className="format">
-            <div>
-              <span>{summary.format}</span>
+          <div className="body">
+            <div className="specs">
+              <span className="price">${summary.marketValEst}</span>
+              <span>{`${summary.numBd}bd`}</span>
               <span> | </span>
-              <span>
-                {this.hasAbestimate()}
-              </span>
-            </div>
-            <div>
-              <span>
-                {`Est. payment ${summary.estPayment}/mo.`}
-              </span>
-              <span>
-                Get pre-qualified
-              </span>
+              <span>{`${summary.numBa}ba`}</span>
+              <span> | </span>
+              <span>{`${summary.sqft} sqft.`}</span>
+              <div>
+                {summary.address}
+              </div>
+              <div className="format">
+                <div>
+                  <img src={circle} alt="forSaleCirc" height="12" />
+                  <span>For sale</span>
+                  <span> | </span>
+                  <span id="abestimate">
+                    Abestimate
+                  </span>
+                </div>
+                <div>
+                  <span>
+                    {`Est. payment ${summary.monthlyPayment}/mo.  `}
+                  </span>
+                  <span>
+                    Get pre-qualified
+                  </span>
+                </div>
+              </div>
             </div>
           </div>
-        </div>
-        <div className="footer">
-          <button type="button" onClick={this.showModal}>contact agent</button>
-          <button type="button" onClick={this.showModal}>take a tour</button>
+          <div className="footer">
+            <button type="button" onClick={() => this.showModal('contact')}>Contact agent</button>
+            <button type="button" onClick={() => this.showModal('tour')}>Take a tour</button>
+          </div>
         </div>
         {modal}
```

Comment:
nice! learned something new here

Reply:
I wonder if there's a better way to do this, but it did work.